index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <div class="nav-bar">
+            <div class="logo">Charles</div>
+            <img src="images/Union.png" alt="Menu Bar">
+        </div>
+        <div class="content">
+            <article>
```

Comment:
- [ ] it would be better if use the `<header>` tag instead of the `<div>` tag because this is your header and the header should contain the `<nav>` tag and inside the nav, the tag put your hamburger menu in order to have a clean structure and improve your SEO benchmarks so I encourage you to do that

Reply:
Done